js/ui/codeView.js

```diff
@@ -16,8 +17,10 @@ const STATE_APP = 0;
 const STATE_TOOLBOX = 1;
 
 const _CODING_APPS = [
-    'com.endlessm.Helloworld',
-    'org.gnome.Weather'
+    // FIXME: this should be extended with a more complex lookup
```

Comment:
Is this still relevant? Hackybird and Hackyballs don't exist anymore, and Dinosaurs is not hackable in the final product. Maybe we can drop the commit that this came from?